docker/ubuntu/Dockerfile

```diff
@@ -0,0 +1,44 @@
+ARG version=20.04
```

Comment:
If that's the default value, then shouldn't it be 16.04?

Reply:
I've changed the default value to be `version=16.04`.